cli/src/commands/developer/scan.rs

```diff
@@ -32,30 +35,43 @@ use std::{
     str::FromStr,
     sync::Arc,
 };
+use ureq::http::Uri;
 use zeroize::Zeroize;
 
 const MAX_BLOCK_RANGE: u32 = 50;
 
 /// Scan the snarkOS node for records.
-#[derive(Debug, Parser, Zeroize)]
+#[derive(Debug, Parser)]
+#[clap(
+    // The user needs to set view_key, private_key, or dev_key,
+    // but they cannot set private_key and dev_key.
+    group(clap::ArgGroup::new("key").required(true).multiple(true))
```

Comment:
It might be simpler to just disallow multiple uses. For instance, why allow using `private_key` AND `view_key`?